.github/workflows/need_reply_remove.yml

```diff
@@ -12,6 +14,7 @@ jobs:
       github.event.comment.author_association != 'OWNER' &&
       github.event.comment.author_association != 'COLLABORATOR' &&
       github.repository_owner == 'pybamm-team'
+      !github.event.issue.pull_request
```

Comment:
This seems to be missing an AND operator (`&&`) in the previous line. Also, I would suggest using checking for the event name instead – something like `github.event_name != 'pull_request'` should work.

Reply:
whoops sorry for that I will add it also regarding the event name I referred to https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment to make it not run on PR comments. Though I could change the trigger to the event name itself.

Reply:
Thanks – your solution might need to be wrapped with `${{ ... }}`, i.e., curly brackets? Yes, both of these ways would be equivalent, I think.

Reply:
There's a style issue related to whitespaces in `need_reply_remove.yml` as well which also should be fixed with it.